tensorflow/go/graph.go

```diff
@@ -61,6 +61,13 @@ type GraphImportOptions struct {
 	// Execution device
 	Device string
 
+	// InputMapping defines a mapping between Outputs in the graph
+	// and Outputs they should be replaced with.
+	InputMapping map[struct {
```

Comment:
I'm not sure what the best practice is here. I made it an anonymous struct to avoid adding API but it's also kind of awkward.

Reply:
Fixed API.